climada/entity/exposures/litpop/litpop.py

```diff
@@ -69,16 +68,21 @@ class LitPop(Exposures):
     """
     _metadata = Exposures._metadata + ['exponents', 'fin_mode', 'gpw_version']
 
-    def set_countries(self, countries, res_arcsec=30, exponents=(1,1),
-                      fin_mode='pc', total_values=None, admin1_calc=False,
-                      reference_year=DEF_REF_YEAR, gpw_version=GPW_VERSION,
-                      data_dir=SYSTEM_DIR):
+    def set_countries(self, *args, **kwargs):
+        """This function is deprecated, use LitPop.from_countries instead."""
+        LOGGER.warning("The use of LitPop.set_countries is deprecated."
+                       "Use LitPop.from_countries instead.")
+        self.__dict__ = LitPop.from_countries(*args, **kwargs).__dict__
+
+    @classmethod
+    def from_countries(cls, countries, res_arcsec=30, exponents=(1,1),
```

Comment:
In this case, I decided to rename `set_countries` to `from_countries`. The prefix `from_` generally indicates that this is a constructor method (cf. `DataFrame.from_dict` in the pandas package).

However, you don't need to adhere to the original `set_`-method name so strictly. For example, `from_country_list` would have been an alternative in this case.